cluster/addons/addon-manager/Makefile

```diff
@@ -15,48 +15,31 @@
 IMAGE=gcr.io/google-containers/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v5.2
-KUBECTL_VERSION?=v1.5.0-alpha.1
+VERSION=v6.0
+KUBECTL_VERSION?=v1.5.0-beta.1
 
-# amd64 and arm has "stable" binaries pushed for v1.2, arm64 and ppc64le hasn't so they have to fetch the latest alpha
-# however, arm64 and ppc64le are very experimental right now, so it's okay
 ifeq ($(ARCH),amd64)
-	BASEIMAGE?=python:2.7-slim
+	BASEIMAGE?=bashell/alpine-bash
```

Comment:
Only Addon Manager v6.0-alpha.1 is using it. It is one of the most popular docker images for bash I could found from docker hub.

I wanted to make this docker image myself, but found [this dockerfile](https://hub.docker.com/r/bashell/alpine-bash/~/dockerfile/) basically does what I want. Hence I adopted it.

One another option would be docker's [official bash image](https://hub.docker.com/_/bash/), which is also using alpine as base image. Not sure if this one is better, its dockerfile seems to have more than we need.